src/main/java/com/conveyal/gtfs/loader/ReferenceTracker.java

```diff
@@ -21,6 +23,7 @@
  */
 public class ReferenceTracker {
     public final Set<String> transitIds = new HashSet<>();
+    public final TreeMultimap<String, String> uniqueValuesForFields = TreeMultimap.create();
```

Comment:
Why is TreeMultiMap used here instead of HashMultiMap? It seems that the TreeMultiMap is particularly useful when the values of the collection of associated with each key should be sorted. It does not appear that there is a need for sorted collections by anything that uses this variable, so it seems the HashMultiMap might be a better choice here.

Reply:
Updated, however, I think this came from a suggested update from @landonreed so maybe a valid reason for using `TreeMultiMap` instead?